taxi/models.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    class Meta:
+        verbose_name = 'manuf'
+        verbose_name_plural = "manufacturers"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        "Manufacturer", on_delete=models.CASCADE, related_name="cars"
+    )
+    drivers = models.ManyToManyField(
+        settings.AUTH_USER_MODEL, related_name="cars"
+    )
+
+    class Meta:
+        verbose_name = "car"
```

Comment:
Ok!
I thought, it had to be added because it was mentioned here https://github.com/AlexTsikhun/py-taxi-service-startproject/blob/master/checklist.md#:~:text=verbose_name%20and%20verbose_name_plural%20should%20be%20in%20lowercase.